client/js/gdpr/chart.js

```diff
@@ -290,10 +290,10 @@ export class GdprChart extends HTMLElement {
     endCap.setAttribute('class', 'series-run-cap');
 
     startCap.setAttribute('r', '5');
-    startCap.setAttribute('cx', points[0].x + 'px');
+    startCap.setAttribute('cx', points[0].x * this.zoom + 'px');
     startCap.setAttribute('cy', points[0].y + 'px');
     endCap.setAttribute('r', '5');
-    endCap.setAttribute('cx', points[points.length - 1].x + 'px');
+    endCap.setAttribute('cx', points[points.length - 1].x * this.zoom + 'px');
     endCap.setAttribute('cy', points[points.length - 1].y + 'px');
```

Comment:
why is this.zoom only being update for cx, x1, x2 but not y positions?

Reply:
also is this.zoom being updated anywhere? (I don't see it)

Reply:
- the zoom only affects horizontal scaling, not vertical
- `this.zoom` is not updated anywhere, that part is added in #80